src/applications/caregivers/sass/toggleButton.scss

```diff
@@ -0,0 +1,13 @@
+.caregiver-toggleButton {
+  box-sizing: border-box;
+  width: 60%;
+
+  button {
+    padding: 10px 30px;
+    box-sizing: border-box;
+    border: 2px solid #0071bc;
+    border-radius: 5px;
+    background: #fff;
+    color: #0071bc;
+  }
```

Comment:
You shouldn't be creating new button styles: 
- Use existing components: https://design.va.gov/components/buttons
- Or, use a named color variable: https://design.va.gov/utilities/color

Reply:
Removing all this code, thank you for the reference!

Reply:
Removed